openrtb_ext/imp_pulsepoint.go

```diff
@@ -4,6 +4,6 @@ package openrtb_ext
 // PubId/TagId are mandatory params
 
 type ExtImpPulsePoint struct {
-	PubID int `json:"cp"`
-	TagID int `json:"ct"`
+	PubID interface{} `json:"cp"`
+	TagID interface{} `json:"ct"`
```

Comment:
PubID and TagID are expected to be int or string. 
However `interface{}` will allow int, string and any other values 

Should use `StringInt` type
https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/util/jsonutil/stringInt.go#L1-L29

Reply:
updated params to use `StringInt` instead of `interface{}`